README.md

```diff
@@ -1,149 +1,114 @@
+3. Use the `git clone` command to clone this repository to your computer.
+
+## Configure LiteFarm applications
+
+The applications are configured with environment variables stored in `.env` files. Configuration information includes secrets like API keys, so the `.env` files should never be added to source control. This repository does contain `.env.default` files for api and webapp. Begin with these, and customize as needed.
+
+1. Create file `packages/api/.env` by copying the `.env.default` file in that folder. 
+
+2. Create file `packages/webapp/.env` by copying the `.env.default` file in that folder. For webapp to work, you must edit your new `.env` file to provide values for two required environment variables:
+     - `REACT_APP_GOOGLE_MAPS_API_KEY` is a Google Maps API key. You should obtain your own key value from [Google](https://developers.google.com/maps/documentation/javascript/get-api-key).
+     - `REACT_APP_WEATHER_API_KEY` is an OpenWeather API key. You should obtain your own key value from [OpenWeather](https://openweathermap.org/api). 
+
+## Runtime setup
+
+### Option 1: Docker containers
+
+This approach runs the LiteFarm applications and the database server in Docker containers. If you prefer to run them directly on your hardware, skip to the next section.
+
+1. Install [docker](https://docs.docker.com/desktop/) and [docker-compose](https://docs.docker.com/compose/install/) if you do not already have them.
+2. In a terminal, navigate to the root folder for the repository.
+3. Execute `docker-compose -f docker-compose.dev.yml up`. Alternatively, you can run `make up` if you have `make` installed.
+    - The initial build will take some time. Subsequent incremental builds are much quicker. 
+    - This process will run [migrations](https://knexjs.org/#Migrations) to set up the PostgreSQL database, then start the Docker containers for webapp, api, databases, and Storybook.
+4. When the build completes, you can load the webapp at `http://localhost:3000`. Storybook is available on `http://localhost:6006`.
+
+### Option 2: Your hardware 
+
+This approach runs the LiteFarm applications and database server directly on your hardware. If you prefer Docker containers, see the previous section.
+
+1. Install PostgreSQL by downloading installers or packages from https://www.postgresql.org/download/. Alternatively, Mac and Linux users can use homebrew as shown below.
+
+   ```bash      
+   # Install homebrew.
+   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+   # Install PostgreSQL.
+   brew install postgresql
+   # Start DBMS service.
+   brew services start postgresql
+   ```
+
+2. Set up the PostgreSQL role (account) and databases for development and test (`pg-litefarm` and `test_farm`, respectively). You will use the `psql` client program. Account setup details will vary by OS. If an installer asks you to choose a password for the `postgres` (superuser) account, use `postgres`.
```

Comment:
It may be worth mentioning that the database names should line up with the env variables set in the api folder (specifically `DEV_DATABASE` and `TEST_DATABASE`)

Reply:
:+1: